File: petclinic-java-gradle-thymeleaf-tailwind-webpack-htmx-ts/src/main/java/dev/pollito/petclinic_java_gradle_thymeleaf_tailwind_webpack_htmx_ts/visit/VisitRepository.java
package dev.pollito.petclinic_java_gradle_thymeleaf_tailwind_webpack_htmx_ts.visit;

import org.springframework.data.jpa.repository.JpaRepository;


public interface VisitRepository extends JpaRepository<Visit, Integer> {

    Visit findFirstByPetId(Integer id);

}

File: petclinic-kt-gradle-thymeleaf-tailwind-webpack-htmx-ts/src/main/kotlin/dev/pollito/petclinic_kt_gradle_thymeleaf_tailwind_webpack_htmx_ts/visit/VisitRepository.kt
package dev.pollito.petclinic_kt_gradle_thymeleaf_tailwind_webpack_htmx_ts.visit

import org.springframework.data.jpa.repository.JpaRepository


interface VisitRepository : JpaRepository<Visit, Int> {

    fun findFirstByPetId(id: Int): Visit?

}
